access/http/handler.go

```diff
@@ -0,0 +1,413 @@
+/*
+ * Flow Go SDK
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package http
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/onflow/flow-go/engine/access/rest/models"
```

Comment:
I'm really sorry for not noticing this sooner, but I don't think we should import `onflow/flow-go` into `flow-go-sdk`. We've intentionally avoided this in the past because it introduces a very large dependency (i.e. the whole Flow protocol) into client applications.

It looks like the REST models are (mostly) generated by Swagger. Would it be possible to regenerate them in this package (i.e. `access/http/models`) or import them from [onflow/flow](https://github.com/onflow/flow/tree/master/openapi/go-client-generated)?

It seems redundant but in this case I actually think it's cleaner to have multiple copies of the models -- one for the server and one for the client. It would also help us confirm the correctness of the OpenAPI/Swagger spec going forward, which means we can be confident that non-Go clients (which do not have the luxury of importing the server-side models) will also be able to consume the API.

Reply:
Ohh you are so right! Thank you I totally missed this. In another PR I also plan to remove ethereum-go dependency.